google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -3733,6 +3769,81 @@ public void testQuerySessionSupport() throws InterruptedException {
     assertEquals(sessionId, statisticsWithSession.getSessionInfo().getSessionId());
   }
 
+  @Test
+  public void testLoadSessionSupportWriteChannelConfiguration() throws InterruptedException {
+    TableId sessionTableId = TableId.of("_SESSION", "test_temp_destination_table_from_file");
+
+    WriteChannelConfiguration configuration =
+        WriteChannelConfiguration.newBuilder(sessionTableId)
+            .setFormatOptions(CsvOptions.newBuilder().setFieldDelimiter(",").build())
+            .setCreateDisposition(JobInfo.CreateDisposition.CREATE_IF_NEEDED)
+            .setSchema(SESSION_TABLE_SCHEMA)
+            .setCreateSession(true)
+            .build();
+    String jobName = "jobId_" + UUID.randomUUID().toString();
+    JobId jobId = JobId.newBuilder().setLocation("us").setJob(jobName).build();
+    String sessionId;
+
+    // Imports a local file into a table.
+    try (TableDataWriteChannel writer = bigquery.writer(jobId, configuration);
+        OutputStream stream = Channels.newOutputStream(writer)) {
+      InputStream inputStream =
+          ITBigQueryTest.class.getClassLoader().getResourceAsStream("sessionTest.csv");
+      // Can use `Files.copy(csvPath, stream);` instead.
+      // Using IOUtils here because graalvm can't handle resource files.
+      IOUtils.copy(inputStream, stream);
+
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+    Job loadJob = bigquery.getJob(jobId);
+    Job completedJob = loadJob.waitFor();
+
+    assertNotNull(completedJob);
+    assertEquals(jobId.getJob(), completedJob.getJobId().getJob());
+    JobStatistics.LoadStatistics statistics = completedJob.getStatistics();
+
+    sessionId = statistics.getSessionInfo().getSessionId();
+    System.out.println(sessionId);
```

Comment:
im guessing this was used for debugging, and meant to be removed?

Reply:
removed.